psibomana/author-quiz
# Add author title

**What does this PR do?**
Add a new author and book title

**Description of Task to be completed?**
Added a page to allow the user to add a new author and / or title

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- Use $ git checkout author-quiz-component-setup to check out the branch containing the changes of this pull request
- Use $ yarn start to start the application
- Visit http://localhost:3000/ to see the updated react application page.
- The titles should be in green in the correct title is selected and in red if the wrong title is selected
- The "Add Author" button below an author image should redirect the user to a new page
- Filling the form and submit should redirect the user to the main page and the newly added book title should be part of the options
